scripts/components/inline-notification/inline-notification.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { icons } from '@eightshift/frontend-libs/scripts/editor';
+import classnames from 'classnames';
+
+export const InlineNotificationType = {
+	warning: 'warning',
+	error: 'error',
+	info: 'info'
+}
```

Comment:
I like this enum solution a lot(at least it reminds me of it), awesome stuff. If this is the case, I believe types should be in CAPS.

```suggestion
export const InlineNotificationType = {
	WARNING: 'warning',
	ERROR: 'error',
	INFO: 'info'
}
```

Reply:
Also I believe since you are able to export it, you can add docs of each type.

```suggestion
/**
* ERROR - when something will fall apart
* WARNING - not falling apart but not good
* ...
*/
export const InlineNotificationType = {
	WARNING: 'warning',
	ERROR: 'error',
	INFO: 'info'
}
```